src/library_webgl.js

```diff
@@ -1307,6 +1307,17 @@ var LibraryGL = {
 #if MAX_WEBGL_VERSION >= 2
               case 0x8919: // GL_SAMPLER_BINDING
               case 0x8E25: // GL_TRANSFORM_FEEDBACK_BINDING
+              //buffer object binding
+              case 0x88ED: // GL_PIXEL_PACK_BUFFER_BINDING
+              case 0x88EF: // GL_PIXEL_UNPACK_BUFFER_BINDING
+              case 0x8C8F: // GL_TRANSFORM_FEEDBACK_BUFFER_BINDING
+              case 0x8A28: // GL_UNIFORM_BUFFER_BINDING
+              case 0x8F36: // COPY_READ_BUFFER_BINDING, same with COPY_READ_BUFFER
+              case 0x8F37: // COPY_WRITE_BUFFER_BINDING, same with COPY_WRITE_BUFFER
+
+              case 0x806A: // GL_TEXTURE_BINDING_3D
+              case 0x8C1D: // GL_TEXTURE_BINDING_2D_ARRAY
+              case 0x8CAA: // GL_READ_FRAMEBUFFER_BINDING
```

Comment:
We should do this for query that returns an opaque object in WebGL 1 and 2. Per spec:
https://www.khronos.org/registry/webgl/specs/latest/2.0/#3.7.2
there's just one missing now 🙂: `GL_DRAW_FRAMEBUFFER_BINDING`

also nit: should sort alphabetically like in the spec and in this code for the WebGL 1 case.

Reply:
Got it. I'll update this patch.

Reply:
Forgot DRAW_FRAMEBUFFER_BINDING was the same as FRAMEBUFFER_BINDING. Thanks for catching that!